src/test/java/com/palantir/docker/compose/connection/ContainerNamesShould.java

```diff
@@ -15,49 +15,56 @@
  */
 package com.palantir.docker.compose.connection;
 
-import static java.util.Arrays.asList;
 import static java.util.Collections.emptyList;
+import static org.hamcrest.Matchers.contains;
 import static org.hamcrest.core.Is.is;
 import static org.junit.Assert.assertThat;
 
+import java.util.List;
 import org.junit.Test;
 
 public class ContainerNamesShould {
 
     @Test
     public void result_in_no_container_names_when_ps_output_is_empty() {
-        ContainerNames names = ContainerNames.parseFromDockerComposePs("\n----\n");
-        assertThat(names, is(new ContainerNames(emptyList())));
+        List<ContainerName> names = ContainerNames.parseFromDockerComposePs("\n----\n");
```

Comment:
Do we need all of these tests? Most of the functionality has moved into `ContainerName` so they're probably replicated there.